<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetMatrix</class>
 <widget class="QDialog" name="DialogSetMatrix">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>259</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioMatrixMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrix Mode</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioTextMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Mode</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>244</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutMatrixMode">
    <item row="0" column="0">
     <widget class="QLineEdit" name="setByte00">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="setByte01">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="setByte02">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="setByte03">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="setByte04">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="setByte05">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="setByte06">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="setByte07">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="setByte08">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="setByte09">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="setByte10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="setByte11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="setByte12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="setByte13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="setByte14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="setByte15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="setByte16">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="setByte17">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="setByte18">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="setByte19">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="setByte20">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="setByte21">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="setByte22">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="setByte23">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="setByte24">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="setByte25">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="setByte26">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLineEdit" name="setByte27">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="setByte28">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="setByte29">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLineEdit" name="setByte30">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLineEdit" name="setByte31">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>25</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="setTextBytes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>01234567890123456789012345678901</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
